test/run_tests.sh

```diff
@@ -33,8 +38,8 @@ run_jm_tests ()
         mkdir -p miniircd
         tar -xzf miniircd.tar.gz -C ./miniircd --strip-components=1
     fi
-    if ! pip install -r ./requirements-dev.txt; then
-        echo "Packages in 'requirements-dev.txt' could not be installed. Exiting."
+    if ! pip install -r ${jm_requirements}; then
```

Comment:
Should surround `${jm_requirements}` in quotes

Reply:
fixed.